docs/manual/cluster-admin/basic-management-operations.md

```diff
@@ -151,4 +151,117 @@ Another example is to restart the whole cluster:
 ./paictl.py service start
 ```
 
-You can use `exit` to leave the dev-box container, and use `sudo docker exec -it dev-box bash` to re-enter it if you desire so. If you don't need it any more, use `sudo docker stop dev-box` and `sudo docker rm dev-box` to delete the docker container.
\ No newline at end of file
+You can use `exit` to leave the dev-box container, and use `sudo docker exec -it dev-box bash` to re-enter it if you desire so. If you don't need it any more, use `sudo docker stop dev-box` and `sudo docker rm dev-box` to delete the docker container.
+
+## <div id="how-to-set-up-https">How To Set Up Https</div>
+
+To configure https certificate for pylon, you need to obtain a digital certificate first, and then save the digital certificate related files in the dev-box container. In the dev-box, you can find the configuration file `services-configuration.yaml`, and then you need to Enter the file path of the saved digital certificate into the `services-configuration.yaml` file. You can choose a self-signed certificate or a certificate issued by a CA. Next, we will first demonstrate the configuration process of a self-signed certificate. The configuration process for the two types of certificates is similar.
+
+### Configure A Self-Signed Certificate
+
+
+#### 1. Enter the dev-box onctainer
+
+You need to use [`paictl`](#pai-service-management-and-paictl) to enter the container in the following way:
+
+```bash
+sudo docker exec -it dev-box bash
+```
+
+#### 2. Create a folder in the dev-box container
+When you enter the container, you need to create a folder and generate a self-signed certificate in this folder. We can create an ssl folder in the home folder.
+
+``` bash
+mkdir /home/ssl
+cd  /home/ssl
+```
+#### 3. Generate RSA private key with openssl
+The next commands you enter will use the FileName parameter multiple times. You can choose a suitable file name to replace FileName.
```

Comment:
commands -> command
```
FileName 
```

->

```
`FileName`
```

Reply:
I think plurals should be used here because there are many commands

Reply:
The following commands? `the next + plural form` is not common.